packages/frontend/index.d.ts

```diff
@@ -26,16 +26,28 @@ interface SimpleLinkType {
 
 interface AdTargetParam {
     name: string;
-    values: string[];
+    value: string | string[];
 }
 
-interface EditionAdTargeting {
-    edition: Edition;
-    paramSet: AdTargetParam[];
+interface EditionCommercialProperties {
+    adTargeting: AdTargetParam[];
+    branding?: Branding;
```

Comment:
This interface enables the Paid Content tone in AMP. It is not currently in use in dotcom-rendering